.github/renovate.json5

```diff
@@ -6,6 +6,7 @@
     "group:allNonMajor",
     ":disablePeerDependencies", 
     "regexManagers:biomeVersions",
+    "helpers:pinGitHubActionDigestsToSemver"
```

Comment:
Do you know how this differs from `pinGitHubActionDigests`? The docs are unclear to me

Reply:
It does the same as `pinGitHubActionDigests`, but it also adds a small comment that shows the version of the pinned action: 

https://docs.renovatebot.com/presets-helpers/#helperspingithubactiondigeststosemver

Example:
```yml
 uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
```

Reply:
Ah, nice